docs/docs-content/release-notes.md

```diff
@@ -11,11 +11,46 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## October 10, 2023 - Release 4.0.19
+
+This release contains the following enhancements and minor bug fixes.
+
+### Enhancements
+
+- You can now modify registry mapping patterns that were previously hardcoded in the Palette CLI and Helm Charts.
```

Comment:
During the helm chart or palette CLI installation 
- You can now modify registry mapping patterns that were previously hardcoded for fetching images from internal oci and docker repositories.
- e.g. In case the harbor internal repo is serving the docker images via pull through cache it can be configured as 

docker.io::teams-jfrog.spectrocloud.dev/v2/image-registry

Reply:
Added: You can now modify registry mapping patterns that were previously hardcoded to fetch images from internal Open Containers Initiative (OCI) and Docker repositories. For example, to pull images from an internal repository serving Docker images, you would configure `docker.io::teams-jfrog.spectrocloud.dev/v2/image-registry`.